### ISSUE TITLE
Add serial port portal

### ISSUE THREAD

**Issue by hadess** [flatpak/xdg-desktop-portal#229]
TBD

**Comment by leamas** [comment]
Usecases includes opencpn [1] (nmea data) and cross-development tools like the Arduino toolbox.

[1] https://www.opencpn.org

**Comment by razzeee** [comment]
I keep running into this, as I somehow end up trying to bundle apps that do something on the serial port.

Examples would be:
https://beta.flathub.org/apps/details/one.flipperzero.qFlipper
https://beta.flathub.org/apps/details/io.github.betaflight.BetaflightConfigurator
https://beta.flathub.org/apps/details/org.expresslrs.ExpressLRSConfigurator

Some can get away with it, if the user attaches the device before starting the flatpak. Others have options to reflash a device, which usually means, that the device will restart and never reappear, as flatpaked apps do not update the serial ports. (https://github.com/flatpak/flatpak/issues/3959#issuecomment-727804466 mentions it only being done on setup)

**Comment by jimmac** [comment]
Apps needing access to external devices through serial interfaces is very common indeed.

**Comment by o-kotb** [comment]
Yes, it's very annoying having to restart KeepassXC flatpak every time you replug a yubikey.
